packages/strapi-plugin-content-type-builder/controllers/ContentTypeBuilder.js

```diff
@@ -122,6 +122,10 @@ module.exports = {
       };
       modelJSON.attributes = formatedAttributes;
 
+      if (mainField) {
+        modelJSON.info.mainField = mainField;
```

Comment:
What happens if the user removed the `info` object? The Lodash `_.set()` method could be a better approach?

Reply:
@Aurelsicoko `modelJSON.info` is created just 2 lines before in the code. No problem about that.